extensions/kafka-client/deployment/src/main/java/io/quarkus/kafka/client/deployment/KafkaProcessor.java

```diff
@@ -377,7 +377,6 @@ private void handleAvro(BuildProducer<ReflectiveClassBuildItem> reflectiveClass,
                     "io.apicurio.registry.serde.DefaultSchemaResolver",
                     "io.apicurio.registry.serde.DefaultIdHandler",
                     "io.apicurio.registry.serde.Legacy4ByteIdHandler",
-                    "io.apicurio.registry.serde.DefaultSchemaResolver",
```

Comment:
What's the reason? Is this class gone?

Reply:
It's a duplicate line, see 4 lines above.